Client/src/components/app-sidebar/AppSidebar.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div
+    class="side-nav"
+    :class="{ minimal: sidebarStore.isMinimal }"
+  >
+    <button
+      class="toggle-sidenav"
+      @click="sidebarStore.toggleIsMinimal"
+    >
+      <span
+        v-if="!sidebarStore.isMinimal"
+        class="icon"
+        >⬅️</span
```

Comment:
We moeten nog een keer nadenken over een goede icon workflow. Voor nu zou ik deze in ieder geval accessible willen maken door er een text aan toe te voegen die voor screen readers leesbaar is (zie de `visually-hidden` utility in `utilities.css`.

Reply:
Daar moeten we zeker nog een keer over nadenken ja. Maar waarom willen we dit icoontje onzichtbaar maken met CSS. Op het moment dat de sidebar is uitgeklapt, is de tekst zichtbaar en is er geen icoon nodig.

Reply:
we willen het icoontje niet onzichtbaar maken, maar een tekst toevoegen voor screen readers, zodat 'sighted' users het icon zien, en screen readers de tekst kunnen voorlezen:

```
    <button
      class="toggle-sidenav"
      @click="sidebarStore.toggleIsMinimal"
    >
      <span
        v-if="!sidebarStore.isMinimal"
        class="icon"
        >⬅️</span
      >
      <span class="visually-hidden">minimaliseer sidebaar</span
      >
    </button>
```